sdk/monitor/monitor-opentelemetry-exporter/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Release History
 
+## (Unreleased)
```

Comment:
Version number missing here.

Reply:
Adding version number with the unreleased tag breaks the check in the pipeline without including a date of release.

Reply:
It should be able to have a version number with unreleased next to it.

Reply:
Looks like not wrapping in parenthesis fixes is.